### PR TITLE
Bump 3.11.4

### PR DESCRIPTION
**Pull request by alberpilot**
Hello team

 Thanks for reviewing the packages 3.11.4 bump.

Best regards, 
Alberto R

### CODE REVIEW

File: rpms/SPECS/3.11.4/wazuh-api-3.11.4.spec [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+%attr(640, root, root) %{_localstatedir}/ossec/api/scripts/wazuh-api.service
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/node_modules
+%attr(750, ossec, ossec) %{_localstatedir}/ossec/api/node_modules/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/node_modules/.bin
+%{_localstatedir}/ossec/api/node_modules/.bin/*
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/logs/api
+%attr(660, ossec, ossec) %ghost %{_localstatedir}/ossec/logs/api.log
+
+%changelog
+* Thu Jan 13 2020 support <info@wazuh.com> - 3.11.4
```
Comment (DFolchA) [inline rpms/SPECS/3.11.4/wazuh-api-3.11.4.spec]: Change this date please

File: rpms/SPECS/3.11.4/wazuh-manager-3.11.4.spec [hunk 2]
```diff
@@ -0,0 +1,1030 @@
...
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap.*
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/template*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content
+%attr(640, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content/*
+
+%{_initrddir}/*
+
+%changelog
+* Thu Feb 13 2020 support <info@wazuh.com> - 3.11.4
```
Comment (DFolchA) [inline rpms/SPECS/3.11.4/wazuh-manager-3.11.4.spec]: Same here.

File: rpms/SPECS/3.11.4/wazuh-api-3.11.4.spec (hunk 1 above)
Comment (alberpilot) [reply rpms/SPECS/3.11.4/wazuh-api-3.11.4.spec]: Done: 20b79cbda984df6fd487ae51afe52f2171648611

File: rpms/SPECS/3.11.4/wazuh-manager-3.11.4.spec (hunk 2 above)
Comment (alberpilot) [reply rpms/SPECS/3.11.4/wazuh-manager-3.11.4.spec]: Done: 20b79cbda984df6fd487ae51afe52f2171648611

**Pull request merged**